src/Bicep.Cli/Services/CompilationService.cs

```diff
@@ -91,6 +91,15 @@ public async Task<Compilation> TestAsync(string inputPath, bool skipRestore)
             var compilation = await bicepCompiler.CreateCompilation(inputUri, this.workspace, skipRestore, forceModulesRestore: false);
             var semantic_model = compilation.GetEntrypointSemanticModel();
 
+            var declarations = semantic_model.Root.TestDeclarations;
+
+            foreach(var declaration in declarations)
+            {
+                declaration.TryGetSemanticModel(out var declarationSemanticModel, out var declarationDiagnostics);
+                var parameters = declaration.DeclaringTest.GetParameters();
```

Comment:
Did you mean to assign these somewhere?

Reply:
Yes.
The idea is to get the parameters as a JToken that we can add to the template evaluation process.  As it was done here:
https://github.com/Azure/bicep/blob/2e3cc1d122bb7e0fff294a90930b4e7b686c5058/src/Bicep.Core.IntegrationTests/TemplateEvaluator.cs#L156C1-L219C10

Reply:
Will that work be completed in a future PR? Maybe in that case the changes to `CompilationService` should be commented out or backed out?

I originally asked about this because the lines added to this class don't appear to have any effect.

Reply:
Seeing that I will use them in the next PR, I will delete them from this one. And add them back in the next one.